nemoguardrails/actions/v2_x/generation.py

```diff
@@ -683,8 +683,8 @@ async def generate_value(
             value = value[:-1]
 
         # Remove variable name from the left if it appears in the result (GTP-4o):
-        if isinstance(prompt, str):
```

Comment:
Let's keep this one in as well. Otherwise, we break support for non-chat models.